install.sh

```diff
@@ -12,7 +17,7 @@ docker pull iptestground/sidecar:edge
 docker run -v ${PWD}:/mount --rm --entrypoint cp iptestground/testground:edge /testground /mount/testground
 
 if [ -z $GITHUB_PATH ]; then
-    echo "Testground was installed to ~/testground. Add this to your path."
+    echo "Testground was installed to \"$TESTGROUND_HOME/testground\". Add this to your path."
 else
-    echo "~/testground" >> $GITHUB_PATH
-fi
\ No newline at end of file
+    echo "$TESTGROUND_HOME" >> $GITHUB_PATH
+fi
```

Comment:
@galargh would you have time to look into this part? It's supposed to run in CI as well.